### DISCUSSION TITLE
imap support

### DISCUSSION THREAD

**Discussion by justin2004** [saulpw/visidata#1909] [Q&A]
i see there is an [imap](https://github.com/saulpw/visidata/blob/develop/visidata/loaders/imap.py) loader.

are there some documented examples using this loader anywhere?

thanks!

**Comment by frosencrantz** [comment]
@justin2004  I do not know, period, but  I would guess you would do something like this:

  `vd "imap://user:password@server/mailbox"`

I don't think there is any more documentation than what is in the source code you linked, and any documents related to the module used and the imap protocol.

- https://www.rfc-editor.org/rfc/rfc2192
- https://docs.python.org/3/library/imaplib.html

Have you tried it yet?  What errors did you see?

**Comment by frosencrantz** [comment]
You have proposed a fix in: https://github.com/saulpw/visidata/pull/1917

Thank you @justin2004.

**Comment by justin2004** [comment]
yup.

thanks for pointing me in the right direction, @frosencrantz !
